src/Libraries/Auth/WebAuth.php

```diff
@@ -136,28 +148,59 @@ public function user()
      * @throws \Exception
      */
 
-    public function verify($code)
+    public function verify($code, $otp_token)
     {
-        if (session()->has($this->authUserKey)) {
+        $user = $this->authService->get($this->keys['otpToken'], $otp_token);
 
-            $user = (array) $this->user();
+        if (new \DateTime() >= new \DateTime($user[$this->keys['otpExpiryIn']])){
+            throw new AuthException(ExceptionMessages::VERIFICATION_CODE_EXPIRY_IN);
```

Comment:
The constant Quantum\Exceptions\ExceptionMessages::VERIFICATION_CODE_EXPIRY_IN was not found.